ports/bsd/bip6.c

```diff
@@ -0,0 +1,489 @@
+/*####COPYRIGHTBEGIN####
+ -------------------------------------------
+ Copyright (C) 2016 Steve Karg
```

Comment:
New files would usually be the author and the date of authorship. However, this file is identical to ports/linux/bip6.c with the exception of 
```#if defined(__APPLE__) || defined(__darwin__)
/* OSX seems not to define these. */
#ifndef s6_addr16
#define s6_addr16 __u6_addr.__u6_addr16
#endif
#ifndef s6_addr32
#define s6_addr32 __u6_addr.__u6_addr32
#endif
#endif
```

and using `en0` instead of `eth0` for the default interface.  

Such a derivative work would keep the original author and date, perhaps adding a second author for the changes if so desired.

Reply:
Thanks for clarifying